tests/conftest.py

```diff
@@ -22,3 +27,24 @@ def index_client(indexd_client):
     https://docs.pytest.org/en/latest/fixture.html#parametrizing-fixtures
     """
     return indexd_client
+
+
+@pytest.fixture()
+def indexd_loader(indexd_client):
+    def load(file_name):
+        docs = []  # type: list[client.Document]
+        with open(file_name, "r") as f:
+            doc_data = json.loads(f.read())
```

Comment:
This is a little easier way to load a json from a file.

```suggestion
            doc_data = json.load(f)
```

Reply:
nice :)